e2e/README.md

```diff
@@ -60,15 +60,28 @@ following environment variables can be exported to customize kubernetes deployme
 - creating kubernetes  cluster
 
 ```console
-$./minikube.sh up
+From the ceph-csi root directory, run:
```

Comment:
This is inside the `console` formatting block, please move it outside of it.

Reply:
:+1: